### PR TITLE
Bring rsyslog.conf in line with other distros

### PR DESCRIPTION
**Pull request by dmcilvaney**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

Our rsyslog config was oddly configured. We were not capturing most log messages (which should go to /var/log/messages), while authentication logs were going to /var/log/messages instead of the normal /var/log/secure.

The log files also had more open permissions than is standard (rw-r--r--) vs (rw-r-----).

###### Change Log  

- Update rsyslog.conf to capture all log messages into /var/log/messages as expected
- Add a filter rule to send auth and authpriv messages to /var/log/secure
- Set all log files to 0640 rather than 0644.

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Local validation of rsyslog function

### CODE REVIEW

**Review by oliviacrain — CHANGES_REQUESTED** [review]
`rsyslog` changes look good to me- please revert the scheduler queue commits that made their way into this PR.

File: toolkit/tools/scheduler/scheduler.go [hunk 1]
```diff
@@ -32,9 +33,11 @@ const (
 // Unlike BuildChannels, schedulerChannels holds bidirectional channels that
 // only the top-level scheduler should have. BuildChannels contains directional channels.
 type schedulerChannels struct {
-	Requests chan *schedulerutils.BuildRequest
```
Comment (PawelWMS) [inline toolkit/tools/scheduler/scheduler.go:35]: I think you've got a "leak" from other changes.;)

Comment (dmcilvaney) [reply toolkit/tools/scheduler/scheduler.go]: Thanks, fixed it.

**Comment by dmcilvaney** [comment]
> `rsyslog` changes look good to me- please revert the scheduler queue commits that made their way into this PR.

Thanks for the catch, fixed it.

**Review by oliviacrain — APPROVED** [review]

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**